module/purpose/Goalie/README.md

```diff
@@ -2,32 +2,41 @@
 
 ## Description
 
-Play soccer in the goalie position. Either bypasses GameController and force playing, or uses GameController information to decide what to do.
+Play soccer in the goalie position.
 
 In the ready state, walks to the goals.
 
-In the playing state (for both normal game mode and penalty shootout), finds the ball, fixates on the ball and dives if the ball is close.
+In the playing state, the goalie will typically stay within the goals unless the ball enters the defending third. In this case, it will either kick it back out (if the closest player) or it will stay between the ball and goals if another played is closer. In penalty states, it will freeze when appropriate and position for attacking if the ball is in the defending third and it is the closest to the ball.
```

Comment:
```suggestion
In the playing state, the goalie will typically stay within the goals unless the ball enters the defending third. In this case, it will either kick it back out (if the closest player) or it will stay between the ball and goals if another player is closer. In penalty states, it will freeze when appropriate and position for attacking if the ball is in the defending third and it is the closest to the ball.
```